Lib/test/test_doctest/test_doctest.py

```diff
@@ -2407,10 +2407,11 @@ def test_DocFileSuite():
        Support for using a package's __loader__.get_data() is also
        provided.
 
-         >>> import unittest, pkgutil, test
+         >>> import importlib.util, unittest, test
```

Comment:
I think that we don't need this test anymore: https://legacy.python.org/dev/peps/pep-0302/

> The `__loader__` attribute must be set to the loader object. This is mostly for introspection and reloading, but can be used for importer-specific extras, for example getting data associated with an importer.

Reply:
Oh yes, I forgot about that invariant. Since the test is actually the same as the one just above, I can just remove the test entirely.